airbyte-ci/connectors/base_images/base_images/bases.py

```diff
@@ -98,4 +102,16 @@ def get_base_container(self, platform: dagger.Platform) -> dagger.Container:
         Returns:
             dagger.Container: The container using the base python image.
         """
-        return self.dagger_client.pipeline(self.name_with_tag).container(platform=platform).from_(self.root_image.address)
+        return (
+            self.dagger_client.pipeline(self.name_with_tag)
+            .container(platform=platform)
+            .from_(self.root_image.address)
+            # Set the timezone to UTC
+            .with_exec(["ln", "-snf", "/usr/share/zoneinfo/Etc/UTC", "/etc/localtime"], skip_entrypoint=True)
+            # Install socat 1.7.4.4
+            .with_exec(["sh", "-c", "apt update && apt-get install -y socat=1.7.4.4-2"], skip_entrypoint=True)
+            .with_exec(["adduser", "--system", "--group", "--no-create-home", self.USER], skip_entrypoint=True)
```

Comment:
```suggestion
            .with_exec(["adduser", "-u", "1000", "--system", "--group", "--no-create-home", self.USER], skip_entrypoint=True)
```

Reply:
We'd like to make the uid/gid consistent with the platform images (where `uid=1000(airbyte)` `gid=1000(airbyte)`)